java-see/build-java-see.sh

```diff
@@ -4,6 +4,15 @@ set -ex
 
 export TERM=${TERM:-dumb}
 
+wget https://download.oracle.com/java/22/latest/jdk-22_linux-x64_bin.tar.gz
```

Comment:
Doesn't the Java buildpack come with the Open JDK? I'm confused as to why this is necessary

Reply:
Oh, so this is running in CI, not CF

Reply:
And I assume we don't want to add Java to the general-task

Reply:
Yep, that is correct on both counts